packages/expo-modules-core/ios/TestUtils/MainThreadInvoker.h

```diff
@@ -0,0 +1,13 @@
+// Copyright 2023-present 650 Industries. All rights reserved.
+
+#ifndef MainThreadInvoker_h
+#define MainThreadInvoker_h
+
```

Comment:
```suggestion
#pragma once

#ifdef __cplusplus
```

- guard with __cplusplus to fix build error for clang module.
- note that the `TestUtils` files will be included in normal build. maybe move the files into `packages/expo-modules-core/ios/Tests/Utils`. or you should update the podspec to only include TestUtils for test_spec